wombat/src/wombat.js

```diff
@@ -4736,7 +4730,7 @@ Wombat.prototype.initPostMessageOverride = function($wbwindow) {
       targetOrigin = this_obj.location.origin;
     }
 
-    // console.log("Sending " + from + " -> " + to + " (" + targetOrigin + ") " + message);
```

Comment:
very minor, but do we want to keep the original syntax to be compatible with ES5, though this is just for debugging and commented out?

Reply:
this line is only for debugging and was changed to accommodate easily adding new things to be logged